### PR TITLE
 Added vehicles seats number and 611 vehicle

### PR DESCRIPTION
**Pull request by Zelyutin**

### CODE REVIEW

File: src/SampSharp.GameMode/World/VehicleModelInfo.cs [hunk 1]
```diff
@@ -267,6 +269,11 @@ private VehicleModelInfo(int type, string name, VehicleCategory category)
         /// </summary>
         public VehicleCategory Category { get; private set; }
 
+        /// <summary>
+        ///     Gets the seats number of this <see cref="VehicleModelInfo" />.
+        /// </summary>
+        public int SeatsNumber { get; private set; }
```
Comment (ikkentim) [inline src/SampSharp.GameMode/World/VehicleModelInfo.cs]: Could you rename this to `SeatCount` along with the parameter in the constructor ?

File: src/SampSharp.GameMode/World/VehicleModelInfo.cs [hunk 2]
```diff
@@ -285,14 +292,14 @@ private VehicleModelInfo(int type, string name, VehicleCategory category)
         /// </summary>
         /// <param name="vehicle">The vehicle to find information about.</param>
         /// <returns>An instance of <see cref="VehicleModelInfo" /> containing information about the specified vehicle.</returns>
-        public static VehicleModelInfo ForVehicle(BaseVehicle vehicle)
+        public static VehicleModelInfo ForVehicle(SampSharp.GameMode.World.BaseVehicle vehicle)
```
Comment (ikkentim) [inline src/SampSharp.GameMode/World/VehicleModelInfo.cs]: Don't think the full namespace qualifier is required here?

Comment (Zelyutin) [reply src/SampSharp.GameMode/World/VehicleModelInfo.cs]: I think it's already fixed in the second commit.

Comment (ikkentim) [reply src/SampSharp.GameMode/World/VehicleModelInfo.cs]: Ah yes

File: src/SampSharp.GameMode/World/VehicleModelInfo.cs (hunk 1 above)
Comment (Zelyutin) [reply src/SampSharp.GameMode/World/VehicleModelInfo.cs]: Done

**Comment by ikkentim** [comment]
Thanks!

**Pull request merged**